readme.md

```diff
@@ -7,7 +7,13 @@ Chat: <a href="https://discord.com/invite/vTvXzBMySh">Discord > Cubes<img src=".
 
 **Why HatTip?**
 
-Instead of writing server code that only works with Express.js, write server code that can be deployed anywhere: AWS, Cloudflare Workers, Vercel, VPS, ...
+With HatTip you write server code that can be deployed anywhere: Cloudflare Workers, AWS EC2, AWS Lambda, Vercel, Netlify, Deno Deploy, Lagon, ...
+
+HatTip prevents vendor locking by abstracting away the underlying platforms. Vercel is getting too expensive? Switch to Cloudflare Workers. Node is becoming too slow? Migrate to Bun. Without changing a single line of application code.
+
+HatTip is designed so that you can still fully tap into the underlying platform. As the need arises at scale, you can focus on a deployment platform and fully leverage its features that make it special.
```

Comment:
Maybe something like this instead:

```suggestion
HatTip lets you use special features of a specific platform. If you do this, it might be harder to move your app to a different platform later. But you can choose to do this if it helps your app work better.
```